updater.sh

```diff
@@ -215,6 +219,6 @@ main() {
   # Check if already running and bail if yes
   flock -n 99 || (echo "Already running script..."; exit 1)
   main
-) 99>/tmp/updater.lock
+) 99>"${LOCKFILE}"
```

Comment:
Is there anything that checks (or should check) that this is a proper file and not a symlink or something?  I'm not sure if there is anything that would prevent (say) overwriting `/etc/passwd` or some such.